authorizations/and/Cargo.toml

```diff
@@ -0,0 +1,54 @@
+[package]
+name = "and"
```

Comment:
Can we think of a better name for this contract? :joy:

Reply:
`logical-and`?  Thought maybe it should be something auth specific, like `must-pass-all` or `authorization-joiner-and` 🤷  open to suggestions